islandora_videojs.module

```diff
@@ -75,6 +75,15 @@ function template_preprocess_islandora_videojs(array &$variables) {
   if (variable_get('islandora_videojs_responsive', TRUE)) {
     drupal_add_css(drupal_get_path('module', 'islandora_videojs') . '/css/videojs_responsive.css', array('group' => CSS_DEFAULT, 'every_page' => FALSE));
   }
+  $object = islandora_object_load($variables['params']['pid']);
+  $variables['transcript'] = FALSE;
+  $transcript_dsid = variable_get('islandora_videojs_transcript_dsid', 'TRANSCRIPT');
+  if ($object[$transcript_dsid]) {
+    $variables['transcript'] = TRUE;
+    $variables['transcript_path'] = url("islandora/object/{$object->id}/datastream/{$transcript_dsid}/view");
+    preg_match('/Language: (.*)/', $object[$transcript_dsid]->content, $matches);
```

Comment:
If... there was indeed XACML preventing access to the datastream, I believe the attempted access here might lead to errors?... just reinforcing the need for the access check, I guess.

Reply:
... also... half-wanting to suggest moving this to a `template_process_islandora_videojs()` implementation, given it's somewhat of a bigger bit of processing to load and regex in the transcript, but... yeah, probably not worth it at this point in time.

Reply:
Sorted